src/Illuminate/Database/Query/Processors/SqlServerProcessor.php

```diff
@@ -21,13 +21,8 @@ public function processInsertGetId(Builder $query, $sql, $values, $sequence = nu
     {
         $connection = $query->getConnection();
 
-        $connection->insert($sql, $values);
-
-        if ($connection->getConfig('odbc') === true) {
-            $id = $this->processInsertGetIdForOdbc($connection);
```

Comment:
This method is now no longer needed? This doesn't break anything?

Reply:
Yea i'm pretty sure my solution will work for ODBC too but i havent tested it.

Reply:
Is there someone who can test this?